src/xcode/ENA/ENA/Source/Views/ENSetting/DescriptionTableViewCell.swift

```diff
@@ -22,10 +22,18 @@ class DescriptionTableViewCell: UITableViewCell, ConfigurableENSettingCell {
 	@IBOutlet var titleLabel: UILabel!
 	@IBOutlet var textView1: UITextView!
 	@IBOutlet var textView2: UITextView!
+	@IBOutlet weak var textView3: UITextView!
 
-	func configure(for _: RiskDetectionState) {
-		titleLabel.text = AppStrings.ExposureNotificationSetting.descriptionTitle
+	func configure(for riskDetectionState: RiskDetectionState) {
+		if riskDetectionState == .disabled {
+			titleLabel.text = AppStrings.ExposureNotificationSetting.descriptionTitleInactive
+		} else {
+			titleLabel.text = AppStrings.ExposureNotificationSetting.descriptionTitle
+		}
+		titleLabel.font = UIFontMetrics.default.scaledFont(for: UIFont.boldSystemFont(ofSize: 22))
```

Comment:
```suggestion
		titleLabel.font = UIFont.preferredFont(forTextStyle: .title)
```

Reply:
Done with https://github.com/corona-warn-app/cwa-app-ios/pull/112/commits/4bbf3f3e49cd921879b33ccb5c710d49b6f53d95 used .title2 to get the right textsize.